### PR TITLE
Verbose Unit Testing for Comprehensive Testing Part 1/3

### PR DESCRIPTION
**Pull request by yctai1994**
Verbose Unit Testing
- killer-sudoku-helper
- hamming
- saddle-points
- pangram (fix)
- leap
- pythagorean-triplet
- collatz-conjecture
- hello-world
- complex-numbers
- raindrops
- nucleotide-count
- rotational-cipher
- difference-of-squares
- secret-handshake
- darts
- rna-transcription
- scrabble-score
- reverse-string
- isogram
- anagram

Type Specification
> Specify the input type to align the output type for easier unit testing.

- saddle-points

### CODE REVIEW

File: exercises/practice/rotational-cipher/runtests.jl [hunk 1]
```diff
@@ -2,53 +2,55 @@ using Test
...
+@testset verbose = true "Rotational Cipher" begin
+    @testset "rotate function" begin
+        @testset "rotate by n" begin
+            @testset "no wrap" begin
+                @test rotate(1, "a") == "b"
+                @test rotate(1, 'a') == 'b'
+                @test rotate(13, "m") == "z"
+                @test rotate(13, 'm') == 'z'
+            end
+            @testset "wrap around" begin
```
Comment (ErikSchierboom) [inline exercises/practice/rotational-cipher/runtests.jl:14]:
```suggestion

            @testset "wrap around" begin
```

File: exercises/practice/rotational-cipher/runtests.jl [hunk 2]
```diff
@@ -2,53 +2,55 @@ using Test
...
-    @testset "full strings" begin
-        @test rotate(5, "OMG") == "TRL"
-        @test rotate(5, "O M G") == "T R L"
-        @test rotate(4, "Testing 1 2 3 testing") == "Xiwxmrk 1 2 3 xiwxmrk"
-        @test rotate(21, "Let's eat, Grandma!") == "Gzo'n zvo, Bmviyhv!"
-        @test rotate(13, "The quick brown fox jumps over the lazy dog.") == "Gur dhvpx oebja sbk whzcf bire gur ynml qbt."
-    end
-end
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+    # Additional exercises                                                        #
```
Comment (ErikSchierboom) [inline exercises/practice/rotational-cipher/runtests.jl:37]: Exercises is a bit of an overloaded term on Exercism. Maybe do:
```suggestion
    # Bonus tests                                                                 #
```

**Review by ErikSchierboom — COMMENTED** [review]
Lovely! Two tiny nits

**Comment by yctai1994** [comment]
> Lovely! Two tiny nits

Done!

**Review by ErikSchierboom — APPROVED** [review]
Great!

**Pull request merged**